rest-api-client with webpack outputs `Module not found: Error: Can't resolve 'path'`

<!-- Thank you for reporting a bug! -->

## Target Package

<!-- Which package do you want to report a bug for? -->

- @kintone/rest-api-client 3.3.0

## Environment

<!-- What environment did you test? -->

- Operating System: macOS 13.2
- Browser:
- Node.js version: 16.16.0

## Current Behavior

https://github.com/kintone/js-sdk-ja/issues/36

Webpack outputs the following error on @kintone/rest-api-client 3.2.0
```
ERROR in ./node_modules/mime-types/index.js 16:14-37
Module not found: Error: Can't resolve 'path' in '/Users/xxx/yyy/node_modules/mime-types'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
        - add a fallback 'resolve.fallback: { "path": require.resolve("path-browserify") }'
        - install 'path-browserify'
If you don't want to include a polyfill, you can use an empty module like this:
        resolve.fallback: { "path": false }
```

This occur occurs by https://github.com/kintone/js-sdk/pull/1894.  This change have added a new dependency [mime-types](https://www.npmjs.com/package/mime-types), which depends on Node.js path package ([see](https://github.com/jshttp/mime-types/blob/master/index.js#L16)).  This problem affects webpack users.  For [UMD](https://unpkg.com/browse/@kintone/rest-api-client@3.2.0/), it works well on the browser because it contains path in the content.

We will fix the problem in the future version.


## Expected Behavior

Webpack build successfully on latest rest-api-client without build configuration changes.

--- a/packages/rest-api-client/package.json
+++ b/packages/rest-api-client/package.json
@@ -63,8 +63,8 @@
     "@rollup/plugin-node-resolve": "^15.0.1",
     "@types/core-js": "^2.5.5",
     "@types/js-base64": "^3.0.0",
-    "@types/mime-types": "^2.1.1",
     "@types/qs": "^6.9.7",
+    "@types/mime": "^3.0.1",
     "rollup": "^2.79.1",
     "rollup-plugin-ecma-version-validator": "^0.2.12",
     "rollup-plugin-node-builtins": "^2.1.2",
@@ -77,7 +77,7 @@
     "core-js": "^3.27.2",
     "form-data": "^4.0.0",
     "js-base64": "^3.7.5",
-    "mime-types": "^2.1.35",
+    "mime": "^3.0.0",
     "qs": "^6.10.5"
   },
   "exports": {

--- a/packages/rest-api-client/src/platform/browser.ts
+++ b/packages/rest-api-client/src/platform/browser.ts
@@ -1,6 +1,6 @@
 import { UnsupportedPlatformError } from "./UnsupportedPlatformError";
 import type { DiscriminatedAuth } from "../types/auth";
-import mime from "mime";
+import mime from "mime/lite";
 
 export const readFileFromPath = (filePath: string) => {
   throw new UnsupportedPlatformError("Browser");

--- a/packages/rest-api-client/src/platform/browser.ts
+++ b/packages/rest-api-client/src/platform/browser.ts
@@ -1,6 +1,6 @@
 import { UnsupportedPlatformError } from "./UnsupportedPlatformError";
 import type { DiscriminatedAuth } from "../types/auth";
-import mime from "mime-types";
+import mime from "mime";
 
 export const readFileFromPath = (filePath: string) => {
   throw new UnsupportedPlatformError("Browser");
@@ -43,7 +43,7 @@ export const buildHeaders = () => {
 export const buildFormDataValue = (data: unknown, fileName?: string) => {
   const options: BlobOptions = {};
   if (fileName) {
-    options.type = mime.lookup(fileName) || undefined;
+    options.type = mime.getType(fileName) || undefined;
   }
 
   return new Blob([data as BlobPart], options);

